scripts/setup/setup-ssh-for-cluster-nodes.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+source scripts/setup/config.sh
+
+rm -f /tmp/sources.list
+cat > /tmp/sources.list <<'END'
+# ubuntu20
+deb http://mirrors.aliyun.com/ubuntu/ focal main restricted
```

Comment:
I believe the CI on GitHub runs on AWS, so there's no need to change the `sources.list`.

Reply:
Yes, we will run CI on GIthub or AWS, but these scripts can all be reused in local testing, and aliyun may be more friendly for Chinese users.